sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -228,6 +236,118 @@ const alwaysBeDeletedRule = "alwaysbedeletedrule";
         }
       });
 
+      it(`Update on non-existent ${entityType} entity throws an error`, async () => {
+        let response;
+        switch (entityType) {
```

Comment:
Have you thought about instead of passing in an array of `EntityType` and performing these case statements in each `it` scope, passing in an array of objects that have the additional fields that your switch statements are using?

For example, something like: 
```js
[{
  entityType: EntityType.QUEUE,
  entityNames: [alwaysBeExistingQueue]
}, {
  entityType: EntityType.SUBSCRIPTION,
  entityNames: [alwaysBeExistingSubscription, alwaysBeExistingTopic]
}, /* etc */
].forEach((testDetails) => {})
```

It just seems like if you're already going through the trouble of reusing an `it` because the testing is similar for each resource, you're losing out on the real benefit since you then check the EntityType in a switch statement and have all your logic in each case. With the way it is now I don't see how this is better than having separate `it` tests that are focused for each resource. (I'd argue it's worse now since resource-specific logic for every resource is in a single `it`, which makes it harder to understand what's going on)

Reply:
Agree with simplifying the switch case within the `it`s - updated